extensions/sequence.md

```diff
@@ -0,0 +1,51 @@
+# Sequence
+
+This extension defines two attributes that can be included within a CloudEvent
+to describe the position of the event relative to the other events from the
+same event source.
+
+The `sequence` attribute represents the value of this event's
+order in the stream of events.  The exact value and meaning of this
+attribute is defined by the `sequenceType` attribute. 
+If the `sequenceType` is missing, or not defined in this specification,
+event consumers will need to have some out-of-band communication with the
+event producer to understand how to interpret the value of the attribute.
+
+## Attributes
+
+### sequence
+* Type: `String`
+* Description: Value expressing the relative order of the event. This enables
+  interpretation of data supercedence.
+* Constraints
+  * REQUIRED
```

Comment:
This may be something we need to add more text on... but the point here is that while the extension itself is optional, IF it is being used then this property is REQUIRED. See the documented-extension.md doc, where it says "Support for any extension is OPTIONAL. When an extension definition uses RFC 2199 keywords (e.g. MUST, SHOULD, MAY), this usage only applies to events that use the extension."

Reply:
Thank you for clarifying.  This makes perfect sense and clears my confusion.